consensus/core/src/block_manager.rs

```diff
@@ -114,69 +114,15 @@ impl BlockManager {
                     missing_blocks.extend(ancestors_to_fetch);
                     continue;
                 }
-                TryAcceptResult::Processed => continue,
+                TryAcceptResult::Processed | TryAcceptResult::Skipped => continue,
```

Comment:
I've made the decision to let received blocks with `block.round <= gc_round` to be allowed received up to here and not straight rejected on the `AuthorityService` in order to make sure that:
* [highest_received_rounds](https://github.com/MystenLabs/sui/blob/c092832b3cc5d0c0eb08ee23ccb50a6af3e1c15f/consensus/core/src/core_thread.rs#L169) keeps getting updated. 
* metrics in the block manager for highest received verified block etc getting updated